packages/sdk/src/utils/chain.ts

```diff
@@ -0,0 +1,113 @@
+import { BigNumber, ethers } from 'ethers'
+import { EthereumProvider } from 'hardhat/types'
+
+import { Chain, L1Chain } from '../interfaces'
+
+export const OptimismKovan = {
+  chainId: Chain.OPTIMISM_KOVAN,
+  chainName: 'OptimismKovan',
+  isTestChain: true,
+  isLocalChain: false,
+  multicallAddress: '0xE71bf4622578c7d1526A88CD3060f03030E99a04',
+  getExplorerAddressLink: (address: string) =>
+    `https://kovan-optimistic.etherscan.io/address/${address}`,
+  getExplorerTransactionLink: (transactionHash: string) =>
+    `https://kovan-optimistic.etherscan.io/tx/${transactionHash}`,
+} as const
+
+export const Optimism = {
+  chainId: Chain.OPTIMISM,
+  chainName: 'Optimism',
+  isTestChain: false,
+  isLocalChain: false,
+  multicallAddress: '0x35A6Cdb2C9AD4a45112df4a04147EB07dFA01aB7',
+  getExplorerAddressLink: (address: string) =>
+    `https://optimistic.etherscan.io/address/${address}`,
+  getExplorerTransactionLink: (transactionHash: string) =>
+    `https://optimistic.etherscan.io/tx/${transactionHash}`,
+} as const
+
+export const OptimismLocal = {
+  chainId: Chain.OPTIMISM_LOCAL,
+  chainName: 'Optimism Local',
+  isTestChain: true,
+  isLocalChain: true,
+  // this is just copy paste
+  multicallAddress: '',
```

Comment:
What do you mean this is just copy/paste? If there are properties on these objects that you don't plan on using, you should delete them

Reply:
They are part of the standard interface that you pass into things like [wallet_addEthereumChain](https://twitter.com/SpankChain/status/1514287825725759491?s=20&t=VNXOnMJ7vlgEi_torhFibA) and used by libraries like useDapp ect.  

I actually looked into this a little bit today and I will remove this since it's not a required property.   

Also, unrelated, but I found that there is a utility that will automatically deploy a multicall contract if one is not already deployed if the frontend ever needs one in testing on a local network or something.